opencensus-shim/src/main/java/io/opentelemetry/opencensusshim/OpenTelemetrySpanImpl.java

```diff
@@ -104,6 +104,7 @@ public void addAnnotation(Annotation annotation) {
 
   @Override
   public void addLink(Link link) {
+    // TODO (jack-berg): Revisit now that adding links after creation is allowed
```

Comment:
this is an odd bit to creep into this PR...  ;)

Reply:
Was lazy 🙂. Removed and opened #6589.